pkg/collect/host_run.go

```diff
@@ -32,16 +38,57 @@ func (c *CollectHostRun) IsExcluded() (bool, error) {
 
 func (c *CollectHostRun) Collect(progressChan chan<- interface{}) (map[string][]byte, error) {
 	runHostCollector := c.hostCollector
+	collectorName := runHostCollector.CollectorName
+	if collectorName == "" {
+		collectorName = "run-host"
+	}
+	var (
+		bundleOutputPath          string
+		bundleOutputRelativePath  string
+		cmdConfigFilePath         string
+		cmdConfigFileRelativePath string
+	)
 
 	cmd := exec.Command(runHostCollector.Command, runHostCollector.Args...)
 
+	cmd.Env = append(os.Environ(),
+		fmt.Sprintf("BUNDLE_PATH=%s", c.BundlePath),
+	)
+	collectorRelativePath := filepath.Join("host-collectors/run-host", collectorName)
+
+	// if we choose to save result for the command run
+	if runHostCollector.OutputDir != "" {
+		bundleOutputRelativePath = filepath.Join(collectorRelativePath, runHostCollector.OutputDir)
+		bundleOutputPath = path.Join(c.BundlePath, bundleOutputRelativePath)
+		cmd.Env = append(cmd.Env,
+			fmt.Sprintf("WORKSPACE_DIR=%s", bundleOutputPath),
```

Comment:
Do we want to create a temp dir somewhere else then copy the data back here? It might be an overkill but again, we would want to ensure that external programs won't be able to access files in the bundle path.

Reply:
Makes sense to avoid giving external program the bundle path and use temp dir.

What do you think about this: instead of let the user specify the outputDir, we only ask them whether their command generate output. 
Then in the documentation we tell the user to ensure their program to utilize the env var `TS_WORKSPACE_DIR` to control where their program writes output to. This way we can avoid the data copying, which could be somehow costly.